modules/get-started/pages/whats-new.adoc

```diff
@@ -6,9 +6,40 @@ This topic includes new content added from version 4.29.0 onwards.
 
 For a full list of product updates, and details of updates prior to version 4.29.0, see the https://github.com/redpanda-data/connect/releases[Redpanda Connect release notes^]. For new Cloud-specific features, see xref:redpanda-cloud:get-started:whats-new-cloud.adoc[].
 
+
+== Version 4.49.1
+
+This release introduces further updates to the xref:components:inputs/postgres_cdc.adoc[`postgres_cdc` input] and xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output].
+
+=== Component updates
+
+This release adds:
+
+- Two new metrics to the xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output]: `snowflake_register_latency_ns` and `snowflake_commit_latency_ns`.
+- An `ignore_nulls` option for schema evolution in the xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output]. 
+
+Deprecated: This release also deprecates the `snapshot_memory_factor` field from the xref:components:inputs/postgres_cdc.adoc[`postgres_cdc` input]. Instead, you must set a batch size, which defaults to `1000` messages. Redpanda recommends upgrading to benefit from this update.
```

Comment:
Is `snapshot_memory_factor` still supported and deprecated, or is it already removed and unsupported? And I don't really understand the "Instead". Do they need to use `snapshot_batch_size` now? Here's a suggestion [consistent with this other Deprecations note](https://deploy-preview-206--redpanda-connect.netlify.app/redpanda-connect/get-started/whats-new/#component-updates-4.) (I didn't see it earlier):
```suggestion
==== Deprecations 

This release deprecates the `snapshot_memory_factor` field from the xref:components:inputs/postgres_cdc.adoc[`postgres_cdc` input]. Now, you must set a batch size, which defaults to `1000` messages. Redpanda recommends upgrading to benefit from this update.
```